### PR TITLE
Blog: Why not discordjs article

### PR DESCRIPTION
**Pull request by Gary50613**

### CODE REVIEW

**Review by HTG-YT — APPROVED** [review]
LGTM.

File: pages/blog/why-not-discordjs.mdx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+![Yeecord Google SERP](@static/blog/why-not-discordjs/yeecord-search-result.png)
+
+## 那我有其他的選擇嗎？
+
+Create-React-App 是個很棒的 React 工具，用簡單的 `react-scripts` 指令就可以跑起 React，而不用擔心後面 babel、webpack 繁雜的設定等等，
+
+<Adsense />
+
+而在時代不斷的更迭到了現在 2023 年，我們有了更多如 vite、NextJS 等的新選擇，在速度與可擴展性上都有大幅的提升，從前端框架的現象我們也可以來觀察一下，JavaScript 是否還有其他解決方案來製作 Discord 機器人。
```
Comment (Nat1anWasTaken) [inline pages/blog/why-not-discordjs.mdx:132]: Not really needed, but I think capitalizing Vite would make it look better

File: pages/blog/why-not-discordjs.mdx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+可以看到除了 discord.js 之外，其他套件也不惶多讓，下面將來講一下我們觀察到的結果。
+
+![Discord bot library in JavaScript](@static/blog/why-not-discordjs/discord-bot-library-javascript.png)
+
+### Discordeno
+
+![Discordeno](@static/blog/why-not-discordjs/discordeno.png)
+
+[Discordeno](https://github.com/discordeno/discordeno) 是使用 deno 編寫的 Discord API 套件，並且在 NodeJS 也可以使用，從他們的 GitHub 首頁來看，上面列舉了許多讓人心動的優勢，讓我們能夠信任並且把機器龍轉移過去：
```
Comment (Nat1anWasTaken) [inline pages/blog/why-not-discordjs.mdx:146]: Same reason as Vite, I think capitalizing will make it look better.

**Review by Nat1anWasTaken — CHANGES_REQUESTED** [review]
Requested some minor changes that is not very important.

File: pages/blog/why-not-discordjs.mdx (hunk 2 above)
Comment (Gary50613) [reply pages/blog/why-not-discordjs.mdx]: fixed

**Review by Nat1anWasTaken — APPROVED** [review]
LGTM

**Pull request merged**